drivers/modem/hl7800.c

```diff
@@ -255,7 +255,7 @@ static const struct mdm_control_pinconfig pinconfig[] = {
 
 #define MDM_MAX_DATA_LENGTH 1500
 #define MDM_MTU 1500
-#define MDM_MAX_RESP_SIZE 128
+#define MDM_MAX_RESP_SIZE 256
```

Comment:
This creates quite high pressure to stack usage, why do we need such a large value here?